src/SuperSelectField.js

```diff
@@ -250,7 +250,10 @@ class SelectField extends Component {
       const updatedValues = selectedItemExists
         ? selectedItems.filter(obj => !areEqual(obj.value, selectedItem.value))
         : selectedItems.concat(selectedItem)
-      this.setState({ selectedItems: updatedValues })
+      this.setState({ selectedItems: updatedValues }, () => {
```

Comment:
Using Arrow functions gives error when building using UglifyJs
```
Unexpected token: punc ()) [./~/material-ui-superselectfield/lib/SuperSelectField.js:379,59][static/js/main.eeb8d3aa.js:
41152,62]
```
Changing to `function() {` does the trick.